app/views/statistics/show.html.erb

```diff
@@ -10,6 +10,16 @@
       <%= t('statistics.dashboard.header.total_types', count: @statistics_dashboard.items.by_type.count) %>
     </p>
   </div>
+
+  <div class="jumbotron col-4-md">
```

Comment:
Is `col-4-md` the right selector here?

Reply:
Not to answer for @jkeck but I did the mockup of this page as HTML and I believe I used `col-4-md` and it worked fine. There will eventually be three (and only three) boxes. And I proposed to hide the boxes below `md` (they're nice, but not critical). But if you have a reason you think `col-4-md` isn't the best choice I'd be interested to hear it, for my own edification...

Reply:
Yeah, this was copy/pasted from the HTML mockup, but I believe what @cbeer is referring to is that the bootstrap column definition would be `col-md-4` (the column width and the breakpoint are swapped).

So it looks like we're not getting any column definition currently and just getting what `justify-content-around` gives us on the jumbotrons.

The main difference being what width we get on the columns since anything below `md` gets hidden anyway.

## Without a col
<img width="980" alt="no-col" src="https://user-images.githubusercontent.com/96776/69198667-4a190480-0aea-11ea-975f-50c6eff49e55.png">

## Using col-md-4
<img width="1031" alt="col-md" src="https://user-images.githubusercontent.com/96776/69198668-4a190480-0aea-11ea-882f-27785356f658.png">

Reply:
Actually, using col-md-4 w/ 3 jumbtrons doesn't look so good, so I'm pulling the class.

<img width="1042" alt="yikes" src="https://user-images.githubusercontent.com/96776/69200023-c4e41e80-0aee-11ea-96d8-fed315d8b28e.png">

Reply:
Ah okay, didn't notice that I transposed those pieces. I just double-checked my HTML mockup and it looks fine with `col-4-md` but that's just because `col-4-md` is meaningless and it's being ignored; my mistake!